src/utils.js

```diff
@@ -868,9 +868,9 @@ export function matchDomain(pattern : DomainMatcher, origin : DomainMatcher) : b
 
 export function stringifyDomainPattern(pattern : DomainMatcher) : string {
     if (Array.isArray(pattern)) {
-        return `(${ pattern.join(' | ') })`;
+        return `RegExp(${ pattern.join(' | ') })`;
```

Comment:
According to the `DomainMatcher` type, if `pattern` is an array, it's an array of strings.

Reply:
(So I think we should just keep this as `(${ pattern.join(' | ') })`)